src/Hashable.php

```diff
@@ -26,7 +26,7 @@ function hash();
      * Determines if two objects should be considered equal. Both objects will
      * be instances of the same class but may not be the same instance.
      *
-     * @param $obj An instance of the same class to compare to.
+     * @param $obj - An instance of the same class to compare to.
```

Comment:
Why this `-`?

Reply:
There is no type after @param so IDE such as Eclipse PDT would raise a syntax error in the doc block. However, I confess I was not sure what type to put there at the time when I was trying to fix it.
I checked further now and supposed $obj could be everything, according to https://github.com/php-ds/ext-ds/issues/140.  Do you think it's better to be 'mixed' there instead?